applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1769,9 +1769,9 @@ void SmallStrainUPwDiffOrderElement::CalculateAndAddCouplingMatrix(MatrixType& r
     for (unsigned int i = 0; i < StressTensorSize; ++i)
         VoigtVector[i] = 1.0;
 
-    Matrix CouplingMatrix = PORE_PRESSURE_SIGN_FACTOR * rVariables.BiotCoefficient * rVariables.BishopCoefficient *
-                            prod(trans(rVariables.B), Matrix(outer_prod(VoigtVector, rVariables.Np))) *
-                            rVariables.IntegrationCoefficient;
```

Comment:
This seems to be a different order of operations than the utility function. Did you check if the result is the same between these two implementations?

Reply:
I ( Wijtze Pieter ) have checked the operations. The order can of the inner product and outer product can be chosen either way, they result in the same dimensions.

Reply:
Thank you Wijtze-Pieter! I put this line into tests and the results are identical for both ways.